src/gl/deferred.cpp

```diff
@@ -291,7 +296,7 @@ uint32_t gl_defer_sorted_draw(uint32_t primitivetype, uint32_t vertextype, struc
 }
 
 // draw deferred models
-void gl_draw_deferred(bool isDrawOpaqueOnly)
+void gl_draw_deferred(int draworder)

```

Comment:
Same as stated above

Reply:
Changed it to use the Enum as parameter.